### ISSUE TITLE
Convert post links into clickable links for in-game reading

### ISSUE THREAD

**Issue by uncovery** [uncovery/uncovery_me#304]
We should use the new tellraw features to make links in posts clickable in-game.

**Comment by LukeBillington** [comment]
Wasn't sure if this was intended for the vote command, but the links there no longer click.

**Comment by uncovery** [comment]
No, it's not, its when there is a new blog post that contains a link. Will create a new issue for the vote command.
